frontend/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "justspaces",
+  "version": "0.0.0",
+  "description": "Frontend assets for the Just Spaces app.",
+  "main": "static/frontend/index.js",
+  "repository": "https://github.com/datamade/just-spaces",
+  "author": "DataMade",
+  "license": "MIT",
+  "private": false,
+  "devDependencies": {
+    "jasmine": "^3.3.1",
+    "jshint": "^2.10.2",
+    "parcel-bundler": "^1.12.2"
+  },
+  "scripts": {
+    "start": "parcel watch src/js/index.js --out-dir static/frontend/js --public-url /static/frontend/js --no-hmr",
+    "test": "parcel build tests/frontend/*.js --out-dir tests/compiled --no-source-maps && jasmine tests/compiled/*.js",
+    "lint": "jshint src/js/*.js tests/frontend/*.js",
+    "build": "parcel build src/js/index.js --out-dir static/frontend/js --public-url /static/frontend/js"
```

Comment:
Here are the custom scripts I've defined so that we can run e.g. `yarn build` (or more accurately, `docker-compose run --rm frontend build`) and get a production build without having to specify a super long command.

Reply:
Again, to satisfy my curiosity: do you have a recommended resource for learning about this approach?